src/pudl/transform/eia.py

```diff
@@ -1318,9 +1317,12 @@ def finished_eia_asset_factory(
     )
     def finished_eia_asset(**kwargs) -> pd.DataFrame:
         """Enforce PUDL DB schema on a cleaned EIA dataframe."""
-        df = convert_cols_dtypes(kwargs[_core_table_name], data_source="eia")
-        res = Package.from_resource_ids().get_resource(table_name)
-        return res.enforce_schema(df)
+        res = PUDL_PKG.get_resource(table_name)
+        return (
+            PUDL_PKG.encode(kwargs[_core_table_name])
```

Comment:
Ensures that all of the EIA core data tables which aren't produced by the harvesting process also get encoded.